Sources/Jetworking/Client/ResponseHandler.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+
+enum ResponseHandler {
```

Comment:
What do you think about transforming the ResponseHandler to a class and initializing it with a `Configuration` once? The `responseHandler` would be a property of the client. Then we could reduce the amount of function parameters. I don't see the advantage of using static functions here.

Reply:
`ResponseHandler` is now a property of `Client`